source/draw/gpu/opengl/backend/EglDisplayContext.ooc

```diff
@@ -56,6 +56,8 @@ EglDisplayContext: class {
 		this _generateContext(sharedContext ? sharedContext _eglContext : null, chosenConfig)
 	}
 	free: override func {
+		if (eglGetCurrentContext() == EGL_NO_CONTEXT)
+			Debug error("Trying to unbind OpenGL context from a thread without a context!")
```

Comment:
Maybe there is a way to also check if it is the same context as the one being freed but if it fails then we get false positives.